samples/subsys/usb_c/sink/src/main.c

```diff
@@ -13,8 +13,8 @@
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(main, LOG_LEVEL_DBG);
 
-#define PORT1_NODE DT_NODELABEL(port1)
-#define PORT1_POWER_ROLE		DT_ENUM_IDX(DT_NODELABEL(port1), power_role)
+#define PORT1_NODE		DT_ALIAS(usbc_port0)
```

Comment:
Update the macro naming to match the alias nodename.  You could also drop the "0" suffix, but I'm fine with or without the suffix.
```suggestion
#define USBC_PORT0_NODE		DT_ALIAS(usbc_port0)
```

Reply:
I think I prefer to keep the 0 in the alias at least. There are boards out there with multiple USB-C ports, so I think it is conceivable that we will have examples using two ports.